raster/r.texture/r.texture.html

```diff
@@ -276,8 +276,3 @@ <h2>AUTHORS</h2>
 M. Ceccarelli - Facolta di Scienze, Universita del Sannio, Benevento<br>
 Markus Metz (correction and optimization of the initial version)<br>
 Moritz Lennert (documentation)
-
```

Comment:
Shouldn't there be one line space (line 273), between the `<h2> AUTHORS </h2>` header and the authors, or it doesn't matter?

Reply:
Of course... with 578 files to edit I have overlooked stuff... will fix it.